test/fixtures/es-modules/json-with-comments/comments.jsonc

```diff
@@ -0,0 +1,6 @@
+{
+    "key": "value"
+    // Some comment
+    /* Also
+    a comment */
+}
```

Comment:
nit... looks like some linting issues with these files (among other things there should be a new-line at the end.